Installation/changes.html

```diff
@@ -155,6 +155,19 @@ <h2 id="release4.10">Release 4.10 </h2>
 <!-- Mesh Generation -->
 <!-- Surface Reconstruction -->
 <!-- Geometry Processing -->
+<h3>Random Point Generators</h3>
```

Comment:
In the manual, the name of that chapter is "Geometric Object Generators", and it is in the "Support Library" part.
http://doc.cgal.org/latest/Manual/packages.html#PkgGeneratorsSummary